index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="eng">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <Title>Portfolio</Title>
+        <link rel="stylesheet" type="text/css" href="index.css">
+    </head>
+
+    <body>
+      <header>
+        <nav class="header">
+          <a href="#"><h2 class="my-logo">My Logo</h2></a>
+          <img src="./logos/Menu.png" alt="Menu">
+        </nav>
```

Comment:
- [ ] [OPTIONAL] You did a great job applying the correct semantic structure. I think both the button and the text look bigger when compared to the design. Please reduce their size to match the design.

Reply:
Thank you very much